From 15aef6e666ee2eb0befa153d277d47754f3656e4 Mon Sep 17 00:00:00 2001
From: Thomas Monjalon <thomas.monjalon@6wind.com>
Date: Thu, 17 Jul 2014 10:30:52 +0200
Subject: [PATCH] examples/pipeline: build with all examples

Upstream-Status: Backport
Imported patch from: http://dpdk.org/browse/dpdk/log/

When adding this packet framework sample (commit 77a3346),
it has been forgotten to add it into the global makefile for
"make examples".

Signed-off-by: Thomas Monjalon <thomas.monjalon@6wind.com>
(cherry picked from commit a6664a09a7caa5e63f9ae625cf1946b0eef7794e)
Signed-off-by: Sreeju Selvaraj <sreeju.armughanx.selvaraj@intel.com>
---
 examples/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/examples/Makefile b/examples/Makefile
index 885c938..1788d6e 100644
--- a/examples/Makefile
+++ b/examples/Makefile
@@ -43,6 +43,7 @@ DIRS-$(CONFIG_EXAMPLE_DPDK_QAT) += dpdk_qat
 endif
 DIRS-y += exception_path
 DIRS-y += helloworld
+DIRS-y += ip_pipeline
 DIRS-y += ip_reassembly
 DIRS-$(CONFIG_RTE_MBUF_SCATTER_GATHER) += ip_fragmentation
 DIRS-$(CONFIG_RTE_MBUF_SCATTER_GATHER) += ipv4_multicast
-- 
1.9.1

